HDF5Examples/JAVA/TUTR/CMakeLists.txt

```diff
@@ -18,7 +18,9 @@ set (CMAKE_JAVA_INCLUDE_PATH ".")
 foreach (CMAKE_JINCLUDE_PATH ${HDF5_JAVA_INCLUDE_DIRS})
   set (CMAKE_JAVA_INCLUDE_PATH "${CMAKE_JAVA_INCLUDE_PATH}${CMAKE_JAVA_INCLUDE_FLAG_SEP}${CMAKE_JINCLUDE_PATH}")
 endforeach ()
-set (CMD_ARGS "-Dhdf.hdf5lib.H5.loadLibraryName=${H5EXAMPLE_JAVA_LIBRARY}$<$<OR:$<CONFIG:Debug>,$<CONFIG:Developer>>:${CMAKE_DEBUG_POSTFIX}>;")
+if (Java_VERSION_STRING VERSION_LESS "24.0.0" OR HDF5_PROVIDES_JNI)
```

Comment:
Should this (and all other occurrences) now be "25.0.0"? Why not just check `HDF5_PROVIDES_JNI` instead of a fragile version check that's replicated in multiple places?

Reply:
yes! I missed that version